### ISSUE TITLE
Installing from github repo: no build folder!

### ISSUE THREAD

**Issue by buddyp450** [STRML/react-grid-layout#86]
The build folder that is being referenced in ```index.js``` is missing when installing from github like: 

```jspm install github:STRML/react-grid-layout```

index.js
```
module.exports = require('./build/ReactGridLayout');
module.exports.Responsive = require('./build/ResponsiveReactGridLayout');
```

**Comment by STRML** [comment]
Yeah, I generally avoid committing built files if I can help it (although I do commit a browser bundle with tags). The `build` folder is published to npm. Try using jspm to get the npm build instead.

**Comment by buddyp450** [comment]
I was pulling the github version because the version on NPM is using a version of react-draggable that's broken with react@0.14.0-beta3... the emptyFunction is now part of the fbjs module. Either way, this issue was fixed in the latest version of react-draggable. If you could just push the versions up I'll be good to go. :)

**Comment by STRML** [comment]
Ah. Figures, that emptyFunction import was pretty dumb.

Unfortunately the master branch of this component is pretty broken right now due to some pending changes in `<Draggable>`. 

I've pushed a few fixes and released as `0.8.5`. That should have 0.14 compat.

**Comment by buddyp450** [comment]
Sorry to bother you again with this but it appears the package has an extra github prefix in the config that's breaking the install:

```
C:\Git\react\client>jspm install npm:react-grid-layout@0.8.5
     Looking up npm:react-grid-layout
     Updating registry cache...
     Looking up npm:deep-equal
     Looking up npm:object-assign
     Looking up npm:react-resizable
     Looking up github:github:strml/react-draggable
     Looking up github:jspm/nodelibs-process
     Looking up github:jspm/nodelibs-fs
     Looking up github:strml/react-draggable

warn Error on lookup for github:github:strml/react-draggable, retrying (1).
     error: The requested URL returned error: 400 while accessing github:strml/react-draggable.git/info/refs?service=git-upload-pack
     fatal: HTTP request failed

     Looking up npm:process

warn Error on lookup for github:github:strml/react-draggable, retrying (2).
     error: The requested URL returned error: 400 while accessing github:strml/react-draggable.git/info/refs?service=git-upload-pack
     fatal: HTTP request failed


warn Error on lookup for github:github:strml/react-draggable, retrying (3).
     error: The requested URL returned error: 400 while accessing github:strml/react-draggable.git/info/refs?service=git-upload-pack
     fatal: HTTP request failed


warn Error on lookup for github:github:strml/react-draggable
     error: The requested URL returned error: 400 while accessing github:strml/react-draggable.git/info/refs?service=git-upload-pack
     fatal: HTTP request failed


err  Error looking up github:github:strml/react-draggable.

warn Installation changes not saved.
```

**Comment by f15gdsy** [comment]
Yeah, have the same problem here using jspm to install from npm.

**Comment by rayshan** [comment]
@STRML 0.8.5 still uses `"react-draggable": "github:strml/react-draggable#v0.3.3"`, which requires multiple things from `react/lib`, so not working with react 0.14.

https://github.com/STRML/react-grid-layout/blob/2c916b3164bffc57d543cd4272f91dde69cf8546/package.json#L37

**Comment by queimadus** [comment]
@rayshan +1
That dependency breaks react@0.14.0 compatibility. 
Specifying a git commit version for npm where this is fixed isn't a option due to the missing build folder.

**Comment by Puffton** [comment]
+1

**Comment by mdeacey** [comment]
+1

**Comment by aghreed** [comment]
+1

Running into the same issue when installing via npm

**Comment by STRML** [comment]
I don't recommend using it (there are still some bugs to work out), but I've published `0.10.0-beta0`, which has proper `build/` and `dist/` folders.
